data/gegno/engines.txt

```diff
@@ -0,0 +1,225 @@
+	"flare sound" "scplasma tiny"
+	description "In order to power the small Halite frame, the Gegno Scin developed this smaller engine system. Based on their larger plasma thrusters, these engines are much more efficient than the Vi's larger, more sluggish Ion Engines, which would never be compact or light enough to run on a fighter."
+
+outfit "SC-12 Plasma Thruster"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 98000
+	mass 25
+	thumbnail outfit/smallscplasma
+	"engine capacity" -25
+	"outfit space" -25
+	thrust 10.6
+	"thrusting energy" 0.8
+	"thrusting heat" 2.7
+	"flare sprite" effect/scplasma/small
+		"frame rate" 0.5
+	"flare sound" "scplasma small"
+	description "Rather than further developing on the Ion Torch design of the Gegno Vi, the Gegno Scin began work on a plasma-based thruster concept. They are more compact, weigh less, and produce significantly more thrust. However, since Ion Torches expel a percentage of heat in their thrust, the more accurate and powerful SC thrusters tend to run notably hotter."
+
+outfit "SC-12 Plasma Steering"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 95800
+	mass 19
+	thumbnail outfit/smallscplasmasteer
+	"engine capacity" -19
+	"outfit space" -19
+	turn 270
+	"turning energy" 0.55
+	"turning heat" 1.8
+	"steering flare sprite" effect/scplasma/small
+		"frame rate" 0.5
+	"steering flare sound" "scplasma small"
+	description "The new SC plasma thruster design excels as a better steering system too, allowing Gegno Scin ships to turn at much faster rates than their opposing brothers. What might take a Gegno Vi ship a long time to maneuver, takes a Scin ship running these only a moment's worth or less."
+
+outfit "SC-14 Plasma Thruster"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 162000
+	mass 45
+	thumbnail outfit/medscplasma
+	"engine capacity" -45
+	"outfit space" -45
+	thrust 26.8
+	"thrusting energy" 2.8
+	"thrusting heat" 7.9
+	"flare sprite" effect/scplasma/med
+		"frame rate" 0.4
+	"flare sound" "scplasma medium"
+	description "SC7-class Thrusters power the Scin's average gunboat or transport, providing a more than reliable speed for a ship in need of it. They easily can propel any medium-sized ship forward as fast as they need with satisfactory system loads."
+
+outfit "SC-14 Plasma Steering"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 156400
+	thumbnail outfit/medscplasmasteer
+	"engine capacity" -34
+	mas 0
+	"outfit space" -34
+	turn 690
+	"turning energy" 1.5
+	"turning heat" 4.78
+	"steering flare sprite" effect/scplasma/med
+		"frame rate" 0.4
+	"steering flare sound" "scplasma medium"
+	description "At a larger scale than typical SC2 thrusters, SC7 steering thrusters are one of the most impactful steering systems in Gegno space, allowing the Scin to reposition their medium and even largest ships many times faster than others."
+
+outfit "SC-15 Plasma Engines"
```

Comment:
Not sure about calling these 'engines' instead of 'thrusters'. The other 'engines' you have have both forward thrust and turn, these only thrust forwards and backwards.

Reply:
Renamed to thrusters. I figured having more than just rear thrust could imply "engines", but you bring up a good point in that that's more fit suitable for ones with steering too, rather than reverse.